src/extension.ts

```diff
@@ -219,7 +263,8 @@ function launchMetals(
   javaPath: string,
   metalsClasspath: string,
   serverProperties: string[],
-  javaOptions: string[]
+  javaOptions: string[],
+  env?: string
```

Comment:
unused?

Reply:
If you've meant env, then it's used now :)